### PR TITLE
adding pt-br translate for flow-control page

### PR DESCRIPTION
**Pull request by ThiagoGMF**
# Changes

- Brazilian portuguese translation for the [Flow Control](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/). page

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @edsoncelio

File: content/pt-br/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -0,0 +1,630 @@
...
+Controlar o comportamento do servidor da API Kubernetes em uma situação de sobrecarga
+é uma tarefa chave para administradores de cluster. O {{< glossary_tooltip
+term_id="kube-apiserver" text="kube-apiserver" >}} tem alguns controles disponíveis
+(ou seja, as _flags_ `--max-requests-inflight` e `--max-mutating-requests-inflight`) 
+para limitar a quantidade de trabalho pendente que será aceito, 
+evitando que uma grande quantidade de solicitações de entrada sobrecarreguem, e
+potencialmente travando o servidor da API, mas essas _flags_ não são suficientes para garantir
+que as solicitações mais importantes cheguem em um período de alto tráfego.
+
+O recurso de prioridade e imparcialidade da API (APF) é uma alternativa que melhora
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/flow-control.md:20]:
```suggestion
O recurso de prioridade e imparcialidade da API ( do inglês *API Priority and Fairness*, APF) é uma alternativa que melhora
```

File: content/pt-br/docs/concepts/cluster-administration/flow-control.md [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+técnica de filas justa para que, por exemplo, um
+{{< glossary_tooltip text="controller" term_id="controller" >}} não precise
+negar as outras requisições (mesmo no mesmo nível de prioridade).
+
+Esse recurso foi projetado para funcionar bem com controladores padrão, que
+usam informantes e reagem a falhas de solicitações da API com exponencial
+back-off, e outros clientes que também funcionam desta forma.
+
+{{< caution >}}
+Solicitações classificadas como "de longa duração" — principalmente relógios — não são
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/flow-control.md:34]: `principalmente relógios` I did get this sentence, what do we mean by `relógios?`

File: content/pt-br/docs/concepts/cluster-administration/flow-control.md [hunk 3]
```diff
@@ -0,0 +1,630 @@
...
+sujeitas ao filtro da prioridade e imparcialidade da API. Isso também é verdade para
+a _flag_ `--max-requests-inflight` sem o recurso da APF ativado.
+{{< /caution >}}
+
+<!-- body -->
+
+## Ativando/Desativando a prioridade e imparcialidade da API
+
+O recurso de prioridade e imparcialidade da API é controlado por um portão de recurso
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/flow-control.md:43]: In other cases we aren't translating `feature gate`

File: content/pt-br/docs/concepts/cluster-administration/flow-control.md [hunk 4]
```diff
@@ -0,0 +1,630 @@
...
+
+## Padrões
+
+Cada kube-apiserver mantém dois tipos de objetos de configuração APF:
+obrigatória e sugerida.
+
+### Objetos de configuração obrigatórios
+
+Os quatro objetos de configuração obrigatórios refletem no
+comportamento de guarda-corpo. Este é o comportamento que os servidores tinham antes
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/flow-control.md:265]: What do you mean by `guarda-corpo`?

File: content/pt-br/docs/concepts/cluster-administration/flow-control.md [hunk 5]
```diff
@@ -0,0 +1,630 @@
...
+que reflete com precisão se os kube-apiservers
+controlam o objeto.
+
+A manutenção também inclui a exclusão de objetos que não são obrigatórios
+nem sugeridos, mas são anotados
+`apf.kubernetes.io/autoupdate-spec=true`.
+
+## Isenção de simultaneidade da verificação de integridade
+
+A configuração sugerida não dá nenhum tratamento especial à saúde
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/cluster-administration/flow-control.md:378]: This sentence can be improved, like `tratamento especial a checagem de saúde das requisições...`

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi, Thanks for your PR! I left a few suggestions below, feel free to ask on slack channel If you need any help :)

**Comment by edsoncelio** [comment]
@devlware PTAL :)
To me it's lgtm

**Comment by Sea-n** [comment]
This PR has no problem, just lack of reviewers.
/remove-lifecycle stale

Could you help this PR move forward? Thanks.
/cc @devlware

**Comment by edsoncelio** [comment]
> This PR has no problem, just lack of reviewers. /remove-lifecycle stale
> 
> Could you help this PR move forward? Thanks. /cc @devlware

Thanks @Sea-n, I reviewed this a few months ago, I'll check it again and ask for help to another reviewer :)

**Pull request merged**